.github/ISSUE_TEMPLATE/documentation_change.yaml

```diff
@@ -0,0 +1,35 @@
+---
+name: 📖 Documentation Change
+description: Suggest an addition or modification to the pynetbox documentation
+labels: ["type: documentation"]
+body:
+  - type: dropdown
+    attributes:
+      label: Change Type
+      description: What type of change are you proposing?
+      options:
+        - Addition
+        - Correction
+        - Removal
+        - Cleanup (formatting, typos, etc.)
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Area
+      description: To what section of the documentation does this change primarily pertain?
+      options:
+        - Features
```

Comment:
These need to be updated, pynetbox documentation has:
* [Endpoint](https://pynetbox.readthedocs.io/en/latest/endpoint.html)
* [Response](https://pynetbox.readthedocs.io/en/latest/response.html)
* [Request](https://pynetbox.readthedocs.io/en/latest/request.html)
* [IPAM](https://pynetbox.readthedocs.io/en/latest/IPAM.html)